packages/client/components/NewMeeting.tsx

```diff
@@ -33,27 +31,42 @@ interface Props {
 
 const MEDIA_QUERY_FUZZY_TABLET = `@media screen and (max-width: ${Breakpoint.FUZZY_TABLET}px)`
 
-const TeamAndSettings = styled('div')<{isDesktop}>(({isDesktop}) => ({
-  alignItems: 'center',
-  display: 'flex',
-  flexDirection: 'column',
-  gridArea: 'settings',
-  marginTop: isDesktop ? 32 : 16,
-  minHeight: 166
-}))
+const TeamAndSettings = styled('div')({
+  marginTop: 16,
+  minHeight: 166,
+  padding: '0px 24px'
+})
 
-const TeamAndSettingsInner = styled('div')({
-  borderRadius: '4px',
-  boxShadow: Elevation.Z1
+const SettingsFirstRow = styled('div')({
+  paddingBottom: 16
+})
+
+const SettingsRow = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  gap: 16,
+  '> div, button': {
+    width: '50%'
+  },
+  [MEDIA_QUERY_FUZZY_TABLET]: {
+    flexDirection: 'column',
+    '> div, button': {
+      width: '100%'
+    }
+  }
 })
 
-const NewMeetingDialog = styled(DialogContainer)<{isDesktop}>(({isDesktop}) =>({
+const NewMeetingDialog = styled(DialogContainer)({
   width: '800px',
-  borderRadius: isDesktop ? Radius.FIELD : 0,
-  minWidth: isDesktop ? 'unset' : '100vw',
-  maxHeight: isDesktop ? 'unset' : '100vh',
-  minHeight: isDesktop ? 'unset' : '100vh',
-}))
+  borderRadius: Radius.FIELD,
+
+  [MEDIA_QUERY_FUZZY_TABLET]: {
```

Comment:
+2 #TIL I like the pattern of handling responsive styles this way, rather than by `isDesktop`